contracts/factory/K1ValidatorFactory.sol

```diff
@@ -66,10 +66,10 @@ contract K1ValidatorFactory is BootstrapUtil, Stakeable {
 
         (bool alreadyDeployed, address account) = LibClone.createDeterministicERC1967(msg.value, ACCOUNT_IMPLEMENTATION, actualSalt);
         BootstrapConfig memory validator = _makeBootstrapConfig(K1_VALIDATOR, abi.encodePacked(eoaOwner));
-        bytes memory _initData = BOOTSTRAPPER._getInitNexusWithSingleValidatorCalldata(validator);
+        bytes memory initData = BOOTSTRAPPER.getInitNexusWithSingleValidatorCalldata(validator);
```

Comment:
was thinking to rename this to accountInitData or accountBootstrapData

Reply:
please check the comment on your branch first and lmk if that makes sense.